modules/heat_conduction/src/base/HeatConductionApp.C

```diff
@@ -69,24 +71,29 @@ void
 HeatConductionApp::registerObjects(Factory & factory)
 {
   registerNamedKernel(HeatConductionKernel, "HeatConduction");
```

Comment:
I'm not touching that Kernel. It is a different part of the residual! Since it is not deprecated I think this is the right registration call.